src/sentry/static/sentry/app/views/organizationStats/projectTable.tsx

```diff
@@ -83,9 +84,14 @@ const StyledProjectTableLayout = styled(ProjectTableLayout)`
   }
 `;
 
-const Percentage = styled(
-  ({children, ...props}) => children !== '' && <div {...props}>{children}</div>
-)`
+type PercentageProps = React.HTMLProps<HTMLDivElement>;
+
+const Percentage = styled(({children, ...props}: PercentageProps) => {
+  if (children === '') {
+    return null;
+  }
+  return <div {...props}>{children}</div>;
+})`
```

Comment:
Wonder if we can simplify this

Reply:
I think so, I'm going to be replacing the chart on this page so I can revisit this at the same time.